ElectronClient/app/theme.js

```diff
@@ -40,6 +40,12 @@ const globalStyle = {
 	raisedBackgroundColor: "#0080EF",
 	raisedColor: "#003363",
 	raisedHighlightedColor: "#ffffff",
+
+	codeColor: "#EFF0F1",
+	codeBorderColor: '#CBCBCB',
+
+	editorTheme: "chrome",
+	highlightTheme: "atom-one-light.css",
```

Comment:
Any reason why it's called "highlightTheme"? In the code, we use the terms "editor" and "viewer" so for consistency I think it would be better to call it "viewerTheme", or maybe "viewerThemeCss" since it's a link to a file.

Reply:
This is the theme specifically for code highlighting, I wanted to avoid confusion by adding that to the name. How do you feel about codeHighlightCss or viewerHighlightCss?

Reply:
Right, I missed that. Yes codeThemeCss would be good in this case.